43/README.md

```diff
@@ -49,19 +47,19 @@ A ZRE *node* represents a source or a target for messaging. Nodes usually map to
 
 ZRE uses UDP IPv4 *beacon* broadcasts to discover nodes. Each ZRE node SHALL listen to the ZRE discovery service which is UDP port 5670 (ZRE-DISC port assigned by IANA). Each ZRE node SHALL broadcast, at regular intervals, on UDP port 5670 a beacon that identifies itself to any listening nodes on the network.
 
-The ZRE beacon consists of one 22-octet UDP message with this format:
+The ZRE beacon consists of one 54-octet UDP message with this format:
 
 ```
-+---+---+---+------+  +------+------+
-| Z | R | E | %x01 |  | UUID | port |
-+---+---+---+------+  +------+------+
++---+---+---+------+  +------+------+------------------+
+| Z | R | E | %x01 |  | UUID | port | CURVE PUBLIC KEY |
++---+---+---+------+  +------+------+------------------+
 
-       Header               Body
+       Header                    Body
```

Comment:
i'm a little in-different to this- but the intent was so:

```
+---+---+---+------+  +------+------+------------------+
| Z | R | E | %x01 |  | UUID | port | CURVE PUBLIC KEY |
+---+---+---+------+  +------+------+------------------+

       Header                    Body
```

appeared "centered" (although looking at it, it may require more ws?) or it can be removed. like i said- indifferent.